src/components/MenuItem.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      hotPrice: 0.2,
+      total: this.props.price
+    };
+  }
+
+  addBread(event) {
+    let breadName = this.props.breads[event.target.value].name;
+    let breadPrice = this.props.breads[event.target.value].price;
+    let total = this.props.price + breadPrice;
+
+    this.setState({
+      bread: breadName,
+      total: !this.state.isHot ? total : total + this.state.hotPrice
```

Comment:
The only issue with handling the price within the `MenuItem` is that various things can affect the price held in state. Price updates on changing bread and selecting hot. You don't know where the changes are coming from or what is updating state. The price could be calculated on the order side of the app when displaying each order item. In this case you'd just send through the id's of the sandwich and bread, as well as whether its hot (true or false)